include/pdal/Utils.hpp

```diff
@@ -149,17 +149,17 @@ namespace Utils
 
     inline std::string tolower(const std::string& s)
```

Comment:
I don't care that much.  I think a cast of the int that was resulting from (std::toupper) would have been fine, too.  Mostly was curious what was wrong.